### DISCUSSION TITLE
Make Langfuse switchable

### DISCUSSION THREAD

**Discussion by MrMojoR** [langfuse/langfuse#1488] [Ideas]
### Describe the feature or potential improvement

It would be great if Langfuse functionality can be turned off. I use the decorators, and if the Langfuse backend is not configured it logs a lot of errors, so I would like to have a parameter, where I can turn off the functionality so I won't get errors when the parameters are not configured. Thanks!

**Comment by marcklingen** [comment]
Agree, we're looking into adding this to all SDKs. Turning this into an issue.
